etl_manager/utils.py

```diff
@@ -79,9 +79,12 @@ def _remove_final_slash(string):
 
 
 # Used by both classes (Should move into another module)
-def _validate_string(s, allowed_chars="_"):
-    if s != s.lower():
-        raise ValueError("string provided must be lowercase")
+def _validate_string(s, allowed_chars="_", allow_upper=False):
+    if allow_upper:
+        pass
+    else:
+        if s != s.lower():
+            raise ValueError("string provided must be lowercase")
```

Comment:
Could you not combine this into a single `if` statement:
```suggestion
    if s != s.lower() and not allow_upper:
        raise ValueError("string provided must be lowercase")
```

Reply:
Yes, that's much nicer.